src/tools/pgflex

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import subprocess
+import sys
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--flex', dest='flex_command', help='flex executable to run', type=str)
+parser.add_argument('--perl', dest='perl_command', help='perl executable to run', type=str)
+parser.add_argument('--builddir', dest='builddir', help='build directory of meson build', type=str)
+parser.add_argument('--privatedir', dest='privatedir', help='temporary directory to avoid conflicts', type=str)
+parser.add_argument('--fix_warning_script', dest='fix_warning_script', help='script for fixing warnings', default='')
+parser.add_argument('-b', dest='no_backup', action='store_true', help='whether no_backup is enabled or not')
+parser.add_argument('-o', dest='output_file', help='output file')
+
+args, rest = parser.parse_known_args()
+rest, input_file = rest[:-1], rest[-1]
+
+# change working directory to temporary directory
+# to avoid conflicts
```

Comment:
maybe add that lex.backup is always named that? Otherwise it's not clear what 'conflict' you could mean.

Reply:
Done.